src/components/content/order/common/ContactDetailsInfo.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { ServiceProviderContactDetails } from '../../../../xpanse-api/generated';
+import YAML from 'yaml';
+import { Button, Popover } from 'antd';
+import React from 'react';
+
+export function ContactDetailsInfo({
```

Comment:
this component is similar to `ContactDetailsText`
We can make it one common component but making button name as a variable. 
Also the CSS must ideally be the same - font size, margin, etc.